etl/steps/data/garden/oecd/2024-07-01/road_accidents.py

```diff
@@ -0,0 +1,149 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import owid.catalog.processing as pr
+import pandas as pd
+from owid.catalog import Origin, Table
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+new_data_cols = [
+    "accident_deaths",
+    "accident_injuries",
+    "accidents_with_injuries",
+    "deaths_per_million_population",
+    "passenger_kms_rail",
+    "passenger_kms_road",
+    "passenger_kms_car",
+    "passenger_kms_bus",
+    "deaths_per_billion_kms",
+]
+
+
+def add_origins(tb: Table, cols: list, origins: Origin) -> Table:
+    for col in cols:
+        tb[col].origins = origins
```

Comment:
you can also use copy_metadata as
```
for col in cols:
    tb[col] = tb[col].copy_metadata(tb["country"])
```

Or any other variable that had not lost the origins.